modules/ingestor/src/graph/sbom/processor/rh_prod_comp.rs

```diff
@@ -103,13 +103,15 @@ impl super::Processor for RedHatProductComponentRelationships {
                 );
             }
 
-            // if it is a top level component, described by the SBOM, packaging components,
-            // then we process those
-            if !((relationship == &Relationship::Generates
-                || relationship == &Relationship::Package)
-                && top_level.contains(prod_node_id))
-            {
-                continue;
+            if !(relationship == &Relationship::Variant) {
```

Comment:
```suggestion
            if relationship != &Relationship::Variant {
```

It feels easier to read.